hubble-be/pom.xml

```diff
@@ -94,25 +94,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
can't remove loader dependency because used by graph-load ui

Reply:
seems we remove common dependency (only keep loader)

Reply:
ok, I see the hugegraph-loader is added at line 72